src/user/components/my_details/details_form.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
```

Comment:
I'm surprised how little this file and, to a lesser extent, the reducer make use of the member view from `admin`. It seems like that would save you a lot of work